docs/pages/deploy-a-cluster/helm-deployments/custom.mdx

```diff
@@ -23,67 +31,72 @@ migrating your setup from a legacy version of the Helm charts.
+You must provide two configurations, one for each node type.
 
-```code
-$ cat << EOF > teleport.yaml
-teleport:
-  log:
-    output: stderr
-    severity: INFO
-
-auth_service:
-  enabled: true
-  cluster_name: custom.example.com
-  tokens:
-  # These commands will generate random 32-character alphanumeric strings to use as join tokens
-  - "proxy,node:$(tr -dc A-Za-z0-9 </dev/urandom | head -c 32)"
-  - "trusted_cluster:$(tr -dc A-Za-z0-9 </dev/urandom | head -c 32)"
-  listen_addr: 0.0.0.0:3025
-  public_addr: custom.example.com:3025
-
-proxy_service:
-  enabled: true
-  listen_addr: 0.0.0.0:3080
-  public_addr: custom.example.com:443
-
-ssh_service:
-  enabled: true
-  labels:
-    cluster: custom
-  commands:
-  - name: kernel
-    command: [/bin/uname, -r]
-    period: 5m
-EOF
-```
+- The `proxy` node configuration should contain at least the `proxy_service` section and the `teleport` section without the `storage` part.
+- The `auth` node configuration should contain at least the `auth_service` section and the `teleport` one.
 
-<Admonition type="tip">
-  You can skip this step if you already have a `teleport.yaml` file locally that you'd like to use.
-</Admonition>
+For the proxy nodes to connect to the auth nodes you must provide a token in their configuration.
+The chart creates a Kubernetes join token named after the release name you can use for a seamless join.
 
-Create the namespace for the config and add the `teleport.yaml` from your local
-disk:
+<Admonition type="warning">
+  When using `scratch` or `standalone` mode, you **must** use highly-available storage (e.g. etcd, DynamoDB, or Firestore) for multiple replicas to be supported.
 
-```code
-$ kubectl create namespace teleport
-$ kubectl --namespace teleport create configmap teleport --from-file=teleport.yaml
-```
+  [Information on supported Teleport storage backends](../../reference/backends.mdx)
 
-<Admonition type="note">
-  The name of the `ConfigMap` used must match the name of the Helm release that you install below (the name just after `helm install`).
-  In this example, it's `teleport`.
+  Manually configuring NFS-based storage or `ReadWriteMany` volume claims is **NOT** supported for an HA deployment and will result in errors.
+</Admonition>
 
-  The name (key) of the configuration file uploaded to your `ConfigMap` must be `teleport.yaml`. If your configuration file is named differently
-  on disk, you can specify the key that should be used in the `kubectl` command:
+Write the following `my-values.yaml` file, and adapt the teleport configuration as you want.
+You can find all possible configuration fields in the [Teleport Config Reference](../../reference/config.mdx).
+
+```yaml
+chartMode: scratch
+
+auth:
+  teleportConfig:
+    # put your teleport.yaml auth configuration here
+    teleport:
+      log:
+        output: stderr
+        severity: INFO
+
+    auth_service:
+      enabled: true
+      cluster_name: custom.example.com
```

Comment:
```suggestion
```

As we automatically set this to the FQDN if it isn't passed here, I recommend we don't tell people to override it. Most of our docs/guides now assume that cluster name == cluster FQDN so it's best to preserve that relationship IMO.